javascript/javascript2/week3/function_as_a_variable/function_main.js

```diff
@@ -0,0 +1,35 @@
+//Function as a variable
+
+//Create an array with 3 items. All items should be functions. Iterate through the array and call all the functions.
+const func1 = () => {
+    console.log('I am function1');
+}
+const func2 = () => {
+    console.log('I am function2');
+}
+const func3 = () => {
+    console.log('I am function3');
+}
+const arr = [func1(), func2(), func3()];
+arr.forEach((func) => {
+    return func;
```

Comment:
when you declare the array like `[func1(), func2(), func3()]` then the array has the value `[undefined,undefined, undefined]` that is because the result of the functions are stored in the array, instead of the functions

this should be declare like `[func1, func2, func3]` so that in the forEach you could have `func()` instead of the `return func` line

there is no return value used here ;)